docs/docsite/rst/dev_guide/testing/sanity/ignores.rst

```diff
@@ -85,6 +85,7 @@ Below are some example skip entries for an Ansible collection::
 
     plugins/module_utils/my_util.py validate-modules!skip # waiting for bug fix in module validator
     plugins/lookup/my_plugin.py compile-2.6!skip # Python 2.6 is not supported on the controller
+    For a full list of sanity tests and test names, see :ref:`all_sanity_tests <https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/index.html#all-sanity-tests>`
```

Comment:
This should be moved outside of the indented examples. It might be enough to move it down a line and remove the indentation.

Reply:
I've already fixed the line of list sanity tests, please have a look at it. Thanks.